### DISCUSSION TITLE
Request: add FAWE/WE //regen parameter to /removeclaim -r

### DISCUSSION THREAD

**Discussion by TomLewis** [GriefPrevention/GriefPrevention#492] [Ideas]
I wish the resotre nature would just use worldedit/fawe and do a //regen.
So have an option that when deleting a claim /removeclaim -r there's a parameter to regen it too (not auto) that would just make that WE selection (fawe allows console based pos1 and pos2 and regen) and do the fawe/WE //regen.

**Comment by 112RG** [comment]
This can be done in my GP addons plugin

**Comment by TomLewis** [comment]
@112madgamer This would be fantastic, is there a list of features in your GP addons plugin?

**Comment by 112RG** [comment]
Yeah, check the GriefPreventionUtilities on my profile. The /edeleteclaims deletes all claims inside your WG region

**Comment by RoboMWM** [comment]
@112madgamer feel free to link/add a short description of your addon in the addons page on the wiki
